.github/workflows/platform_ci_workflow.yml

```diff
@@ -21,13 +21,19 @@ on:
       workspace_name:
         description: "the workspace name hosting the AML instance."
         required: true
-      azure_credentials:
-        description: "the credentials to use to authenticate with Azure."
+      arm_client_id:
```

Comment:
secrets section is not needed here anymore

Reply:
Removed and replaced secrets. with vars.